shared/css/mixins.scss

```diff
@@ -1,7 +1,7 @@
 // Useful mixins, especially for stuff that's browser-prefix heavy.
 // Could always use something like Prefixr for this instead...
 
-@mixin user-select($val) {
+@mixin user-selects($val) {
```

Comment:
Yeah, I'm not sure we ever figured out what the actual build issue was, just worked around it.  I'd like to give it another shot - being able to deduplicate this stuff is why we're using SCSS, right? (Never mind that we should probably be using [Prefixr](http://prefixr.com/) for this...)